### DISCUSSION TITLE
Viewing h5 output files in Paraview

### DISCUSSION THREAD

**Discussion by samerps** [thliebig/openEMS-Project#34] [Q&A Usage] [answered]
Hi, I'm currently learning openEMS and I'm going through the Python tutorials. I was wondering how can view the output h5 files in Paraview? For example, the Sim_Patch Python tutorial generates an nf2ff_E h5 group, and when I try to open them in Paraview it fails to load. 

Many thanks

**Comment by thliebig** [comment, marked as answer]
Hi, paraview cannot open hdf5 files. and are you sure you want to look at these? They contain the near-fields needed for the far-field calculations. I assume you rather want to look at the far-fields? 
I think currently there is no converter for the openEMS data in hdf5 to any vtk format in python.

**Reply by ringo458** [reply]
I wondered about that myself. I think the average user thinks they can view in Paraview because it says that somewhere, then they figure out where the dumps are and just try to view what's there. You download Paraview, possibly for the first time, and try to use the GUI ab initio. When it doesn't work you think you're doing it wrong because the GUI shows you a half dozen h5 readers. Surely one of them must be the right one ....

**Reply by thliebig** [reply]
Yes indeed. For the Octave interface such exports/converters do exist. But not (yet) for python.
But for regular dumps you can (and I guess should) select the vtk as output format. In this case the openEMS engine itself writes vtk files to be viewed in paraview.

**Reply by samerps** [reply]
Many thanks for your reply, I am interested in both near field and far fields
